packages/block-editor/src/components/inserter/preview-panel.js

```diff
@@ -15,8 +16,11 @@ import BlockCard from '../block-card';
 import BlockPreview from '../block-preview';
 
 function InserterPreviewPanel( { item } ) {
-	const { name, title, icon, description, initialAttributes, example } = item;
+	const { name, title, icon, description, initialAttributes } = item;
 	const isReusable = isReusableBlock( item );
+	const hoveredItemBlockType = getBlockType( name );
+	const example = item.example || hoveredItemBlockType.example;
```

Comment:
As far as I know, it is not explicitly stated that the example can be used for block variations.
Should I leave any comments here?

Reply:
It might be worth adding a small comment to explain why you're checking for both, for sure 👍